src/local-history/local-history-manager.ts

```diff
@@ -405,7 +409,7 @@ export class LocalHistoryManager {
             }
         }
         catch (err) {
-            console.log(err);
+            this.outputChannel.appendMessage(OutputChannelSeverity.ERROR, moment().format('hh:mm:ss'), [err]);
```

Comment:
I do not like that each time we call the `appendMessage` we will need to pass the formatted time.
I think that the logger itself should handle the timestamp as to not duplicate the formatting of the time and also for a cleaner and simpler solution.